data/json/items/gunmod/muzzle.json

```diff
@@ -90,15 +88,14 @@
     "location": "muzzle",
     "mod_targets": [ "pistol", "smg", "rifle" ],
     "install_time": 200,
-    "damage_modifier": -4,
-    "loudness_modifier": -50,
+    "loudness_modifier": -80,
```

Comment:
A manufactured suppressor should probably be better at reducing noise than the home-crafted version made from an old oil filter.